js/index.js

```diff
@@ -14,11 +15,20 @@ let responseContent;
 
 let parsed;
 
+input.addEventListener('keyup', ({ keyCode }) => {
+  //used destructuring for event.keyCode => event object is destructured and {keyCode} takes it's value.
+  if (keyCode === 13) {
+    const search = event.target.value;
```

Comment:
Since you destructured "event" argument into { keyCode } only, you won't have access to it anymore.